external-api/src/cronJobs/eStoreCronJobManager.ts

```diff
@@ -1,31 +0,0 @@
-import CronJobManager from 'cron-job-manager';
```

Comment:
Now that this is gone where are we setting up the polling of the tables? This might be that we aren't and I'm just missing something just struggling to find where it would be defined!

Reply:
Polling to the table will be setup in `eStoreApi.ts` once all the PRs have been merged.